packages/teams-js/test/public/pages.spec.ts

```diff
@@ -460,13 +468,15 @@ describe('Testing pages module', () => {
             const promise = pages.navigateToApp(navigateToAppParams);
 
             const executeDeepLinkMessage = utils.findMessageByFunc('executeDeepLink');
-            utils.respondToMessage(executeDeepLinkMessage, true);
-            await promise;
-
-            expect(executeDeepLinkMessage).not.toBeNull();
-            expect(executeDeepLinkMessage.args[0]).toBe(
+            validateExpectedArgumentsInRequest(
```

Comment:
Looks like the order of this was switched (used to happen after the respond / await, and now we validate before). Why the change?

Reply:
(Assume I wrote this for every place where it is switched around)

Reply:
In the original ordering, we were using the `executeDeepLinkMessage` object in the call to `utils.respondToMessage` and THEN checking if it was not null.

With this reordering, we first check if it's null and then we use it.

Functionally this may not be that different -- prior to my reordering, the test would fail (by crashing) when `utils.respondToMessage` tried to use the null `executeDeepLinkMessage` object. The check that `executeDeepLinkMessage` was null was essentially pointless. Now with my reordering, the test will still fail, but it will fail with an error message telling you that `executeDeepLinkMessage` was null as opposed to just crashing somewhere inside of `utils.respondToMessage` where it assumes the message isn't null.